docs/faq.md

```diff
@@ -14,12 +14,12 @@ menu:
 ### Why is the dependency resolution process slow?
 
 While the dependency resolver at the heart of Poetry is highly optimized and
-should be fast enough for most cases, sometimes, with some specific set of dependencies,
```

Comment:
This should be left as it is. The new version describes a single dependency, while the old one (correctly) describes that there are sets of dependencies that cause slow resolution.

Reply:
The original sounded wrong to me, as "some specific set of dependencies" sounds like it's referring to one specific set. I committed a tweaked version which would be a sidegrade from "some specific _sets_ of dependencies"